tide/master/PixelStreamWindowManager.cpp

```diff
@@ -141,6 +141,12 @@ void PixelStreamWindowManager::handleStreamStart(const QString uri)
         return;
     }
 
+    if (uri == "Launcher")
+    {
+        openWindow(uri, QPointF(), QSize(), StreamType::LAUNCHER);
```

Comment:
this is a duplication of the line below! Do:

```
// external streams don't have a window yet, create one now
const auto streamType = _isPanel(uri) ? StreamType::LAUNCHER : ...;
openWindow(uri, QPointF(), QSize(), streamType);
```

then rename the existing _isPanel() helper function to _isLauncher()